src/components/API-calls.js

```diff
@@ -5,7 +5,39 @@ const api = axios.create({
 });
 
 export const getReviewList = () => {
```

Comment:
This hasn't been asked for so you don't need to do it. It's important that you _stick to what you're being asked to do_, because in a job you would get in trouble for this because the the sprint doesn't mention search functionality. Please try and stick to the tickets, and if you finish them, then you can add other functionality.